File: src/main/java/com/example/animalspring/model/Animal.java
package com.example.animalspring.model;


import com.github.javafaker.Faker;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Data
@Entity
@Table(name="animal")
@NoArgsConstructor
public class Animal {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;
    private String name;
    private int no;

    public Animal(String name, int no) {
        this.name = name;
        this.no = no;
    }

    @Override
    public boolean equals(Object o){

        Animal animal = (Animal) o;

        return this.name.equals(animal.name);

    }
}

File: src/client/src/Components/Home/Animal.js
import React, { useEffect } from "react";

function Animal({animal}){

    useEffect(()=>{

        console.log("aici");

    },[]);


    return(


                <tr>
            
                    <td>{animal.id}</td>
                    <td>{animal.name}</td>
                    <td>{animal.no}</td>
    
                </tr>


    );
}

export default Animal;